recipes-rxjs/src/app/components/recipe-creation/recipe-creation.component.html

```diff
@@ -0,0 +1,118 @@
+<div class="recipe-creation-overview">
+  <section class="recipe-creation-overview__header">
+    <h1>Create a new recipe</h1>
+    <p>Fill in the form below to create a new recipe</p>
+  </section>
+  <form [formGroup]="form" (ngSubmit)="onSubmit()" class="recipe-creation-form">
+    <mat-form-field class="form-field">
+      <mat-label>Title</mat-label>
+      <input matInput placeholder="Title" formControlName="title" />
+      <mat-error *ngIf="form.get('title')?.hasError('required')">
```

Comment:
Once you have initialized the form inside the component TS, you can access the form controls as follows: `form.controls.title` for example.